recipes/heavydb/meta.yaml

```diff
@@ -0,0 +1,575 @@
+          else                                                                                # [unix]
+            echo "heavysql not found in PATH(=$PATH)" && exit 1                               # [unix]
+          fi                                                                                  # [unix]
+          source ${PREFIX}/etc/conda/deactivate.d/{{ PKG_NAME }}_deactivate.sh                # [unix]
+          if [[ -x "$(command -v heavysql)" ]]                                                 # [unix]
+          then                                                                                # [unix]
+            echo "Unexpectedly found heavysql in PATH(=$PATH) after deactivation"  && exit 1  # [unix]
+          else                                                                                # [unix]
+            echo "heavysql not in PATH after deactivation"                                    # [unix]
+          fi                                                                                  # [unix]
+          source ${PREFIX}/etc/conda/activate.d/{{ PKG_NAME }}_activate.sh                    # [unix]
+        # Test installation
+        - heavysql -v              # [unix]
+        - which initheavy    # [unix]
+        - which heavydb    # [unix]
+        - heavysql.exe -v          # [win]
+        - |
+          if [ -x "$(command -v nvidia-smi)" ]; then  # [unix and cuda_compiler_version not in (undefined, "None")]
+            mkdir data && initheavy data              # [unix and cuda_compiler_version not in (undefined, "None")]
+            heavydb --version                         # [unix and cuda_compiler_version not in (undefined, "None")]
+            rm -rf data                               # [unix and cuda_compiler_version not in (undefined, "None")]
+          fi                                          # [unix and cuda_compiler_version not in (undefined, "None")]
+        # these are provided by heavydb-common package:
+        - which heavydb_insert_sample_data  # [unix]
+        - which startheavy  # [unix]
+
+    about:
+      home: https://heavy.ai
+      license: Apache-2.0
+      license_family: APACHE
+      license_file: LICENSE.md
+      summary: The HeavyDB database
+
+      description: |
+        HeavyDB is an in-memory, column store, SQL relational database
+        that was designed from the ground up to run on GPUs.
+
+        This recipe provides both CUDA enabled and CUDA disabled heavydb
+        packages.
+      doc_url: https://docs.heavy.ai/
+      dev_url: https://github.com/heavyai/heavydb
+
+  - name: pyheavydb
+    version: {{ version }}
+    build:
+      skip: true  # [cuda_compiler_version in ("11.0",)]
+      noarch: python
+      script: python -m pip install -vv --no-deps python/.
+      run_exports:
+        - {{ pin_subpackage('pyheavydb',  max_pin='x.x.x') }}
+    requirements:
+      host:
+        - pip
+        - python >=3.7
```

Comment:
python 3.7 is the lowest available package on conda-forge

Reply:
Do you mean that the lower pin should be removed?

Reply:
yes, the lowest version on conda-forge is currently 3.7

Reply:
```suggestion
        - python
```